pkgs/by-name/gl/glider/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec{
```

```suggestion
buildGoModule rec {
```

i've never seen a package that didn't put a space after `rec`, probably best to keep formatting in line with the rest of nixpkgs.

---

I think this may not be a problem, https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20rec%7B&type=code . The formatter I use is nixpkgs-fmt.

---

24 matvhes vs the 25k matches of [this search that includes the space](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20rec%20%7B&type=code)

that's 0.1% of the code ommiting the space

and really, technically i should be telling you to use finalAttrs instead of rec

---

> that's 0.1% of the code ommiting the space

I'll add the space later.

> use finalAttrs instead of rec

Is there a way to use `finalAttrs` pattern for `buildGoModule`?

---

hmm, i don't think there is yet, oops.